Source/Core/Core/Movie.cpp

```diff
@@ -1489,4 +1489,4 @@ void Shutdown()
   s_currentInputCount = s_totalInputCount = s_totalFrames = s_tickCountAtLastInput = 0;
   s_temp_input.clear();
 }
-};
+};  // namespace Movie
```

Comment:
Note that the semicolon here can be removed as well, if you're making changes.

Reply:
I didn't actually do this, blame clang-format.